python/semantic_kernel/connectors/ai/open_ai/open_ai_response.py

```diff
@@ -0,0 +1,210 @@
+from typing import AsyncGenerator, Dict, List, Optional, Union
+
+from openai import AsyncStream
+from openai.types import Completion
+from openai.types.chat import ChatCompletion, ChatCompletionChunk
+
+from semantic_kernel.connectors.ai.ai_response import AIResponse
+from semantic_kernel.connectors.ai.open_ai.models.chat.function_call import FunctionCall
+
+
+class OpenAITextResponse(AIResponse):
+    "A text completion response from OpenAI."
+
+    raw_response: Union[Completion, AsyncStream[Completion]]
+    stream_content: Optional[str] = None
+    stream_all_content: Optional[List[str]] = None
+
+    @property
+    def content(self) -> Optional[str]:
```

Comment:
Does this type needs to be Optional?

Reply:
yes, because the content is optional in the openai response as well (for instance when you have a function call instead)